### DISCUSSION TITLE
Welcome to Medical_Imaging_With_Deep_Learning Discussions!

### DISCUSSION THREAD

**Discussion by elee02** [elee02/Medical_Imaging_With_Deep_Learning#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elee02** [comment]
@dilshod-obidov You can write your opinions about our project here

**Reply by dilshod-obidov** [reply]
Got it, thank you
